arcade/arcade/core/output.py

```diff
@@ -14,9 +14,10 @@ def success(
         self,
         *,
         data: T | None = None,
+        warnings: list[ToolCallWarning] | None = None,
     ) -> ToolCallOutput:
         value = getattr(data, "result", "") if data else ""
-        return ToolCallOutput(value=value)
+        return ToolCallOutput(value=value, warnings=warnings)
```

Comment:
What about `logs` of type `ToolCallLog` which contain a `.level`?

Reply:
Ok my initial thought was that we would have `ToolCallOutput(value=value, debugs, logs, warnings)` but your idea makes much more sense. Especially because it would maintain order between types of logs.